jobs/scripts/plips.sh

```diff
@@ -1,17 +1,28 @@
 #!/bin/sh
 # buildout, core and AT tests
-pip install -r requirements.txt
+pip install -Ur requirements.txt
 buildout buildout:git-clone-depth=1 -c {buildout}
 
-return_code="all_right"
 export PATH="/usr/lib/chromium-browser:$PATH"
 export ROBOT_BROWSER=chrome
+export ROBOTSUITE_PREFIX=ROBOT
 
-xvfb-run -a --server-args='-screen 0 1920x1200x24' bin/alltests --xml --all || return_code=$?
-xvfb-run -a --server-args='-screen 0 1920x1200x24' bin/alltests-at --xml || return_code=$?
+alias xvfb-wrap="xvfb-run -a --server-args='-screen 0 1920x1200x24'"
+
+return_code='all_right'
+
+# Archetypes tests without Robot
+xvfb-wrap bin/jenkins-alltests-at --all --xml -t '!ROBOT' || return_code="$?"
```

Comment:
``jenkins-alltest`` and friends are only available on 4.3, 5.0 upwards have plain ``alltests``. As this is the script used for PLIPs and there are only PLIPs for 5.1 and 5.2 right now, this should drop the ``jenkins-`` prefix